_projects/pmefr_project.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: page_project
+title: Programming Model Extensions for Resilience
+date: 2016-02-10
+updated:
+navbar: Projects
+footer: true
+project_url:
+status: starting
+topics:
+  resilience
+keywords:
+  -
+head: amer_a
+members:
+  - balaji_p
+  - beltran_v
+  - casas_m
+---
+{% comment %}
```

Comment:
feel free to delete those Liquid comments (the part between each block of `{% comment %}` and `{% endcomment %}`